src/magick-image.ts

```diff
@@ -1,4 +1,4 @@
-// Copyright Dirk Lemstra https://github.com/dlemstra/magick-wasm.
+﻿// Copyright Dirk Lemstra https://github.com/dlemstra/magick-wasm.
```

Comment:
Can you check if you added or removed a BOM (byte order marker) here and revert that part of the change>